buildSrc/src/main/java/Dependencies.kt

```diff
@@ -11,7 +11,7 @@ object Versions {
     const val coroutines = "1.5.0"
 
     const val junit = "4.12"
-    const val robolectric = "4.1"
+    const val robolectric = "4.6.1"
```

Comment:
By the way, `./gradlew feature-autofill:test` fail when run locally saying that `org.robolectric:android-all:jar:8.1.0-robolectric-4611349` is missing. So I raised version to 4.6.1 and it worked.
Should I revert it back to 4.1 or is it acceptable to use 4.6.1?

Reply:
I've just hit the same locally for some other tests. @pocmo already filed this in https://github.com/mozilla-mobile/android-components/issues/10550